video2numpy/utils.py

```diff
@@ -20,3 +23,42 @@ def handle_youtube(url):
     cv2_vid = f.get("url", None)
     dst_name = info.get("id") + ".npy"
     return cv2_vid, dst_name
+
+# Unfortunately, I did not find a way to convert directly from the url
```

Comment:
can you convert from mp4 -> numpy array with audio?

Reply:
Finally I found that ffmpeg-python worked